assets/css/components/_status.scss

```diff
@@ -0,0 +1,23 @@
+.status {
```

Comment:
`status` is an abstract name. Let's rename this one because it is a component. `status-badge` also works.

Reply:
Updated! Thanks 👍